ui/components/FilterableTable.tsx

```diff
@@ -31,8 +31,10 @@ export type FilterConfigCallback = (v: any) => any;
 
 export const filterByStatusCallback: FilterConfigCallback = (v) => {
   if (v.suspended) return "Suspended";
-  else if (computeReady(v["conditions"])) return "Ready";
-  else return "Not Ready";
+  else if (computeReady(v["conditions"]) === ReadyType.Reconciling)
+    return ReadyType.Reconciling;
+  else if (computeReady(v["conditions"])) return ReadyType.Ready;
```

Comment:
A constant could be used for the conditionsKey instead of hardcoded strings.

Reply:
good upgrade